newbius/installations/example/main.tf

```diff
@@ -88,6 +99,10 @@ module "k8s" {
       id        = submount.id
       mount_tag = submount.mount_tag
     }]
+    accounting_storage = coalesce({
+      id        = module.filestore.accounting_storage.id
+      mount_tag = module.filestore.accounting_storage.mount_tag
+    }, null)
```

Comment:
What do such lines actually do?
- Accessing `module.filestore.accounting_storage.id` and `module.filestore.accounting_storage.mount_tag` will cause error in case accounting is disabled.
- `coalesce` of some data and null will always return null.